cmake/CompilerFlags.cmake

```diff
@@ -27,7 +27,7 @@ if (ENABLE_WARNING_VERBOSE)
 		endforeach()
 
 		set(C_WARNING_FLAGS
-			/Wall
+			/W3
```

Comment:
ok, are these there with `/W4` too? (tend to enable the most useful verbose warning level by default, disable with `-DENABLE_VERBOSE_WARNINGS=OFF`)

Reply:
`/W4` does not cause any warnings for system headers but it causes a lot of `C4324` warnings with `rdpSettings`:

`'rdp_settings': structure was padded due to alignment specifier`

So if we want to enable `/W4` I think we should disable `C4324` - shall I make that change?

Reply:
as we´ve done this for `gcc` and `clang` as well (`-Wno-padded` et al) yes, would be a good idea.

Reply:
@akallabeth updated accordingly 👍